habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -424,8 +424,8 @@ def _eval_checkpoint(
         stats_episodes = dict()  # dict of dicts that stores stats per episode
 
         rgb_frames = [
-            []
-        ] * self.config.NUM_PROCESSES  # type: List[List[np.ndarray]]
+            [] for _ in range(self.config.NUM_PROCESSES)
+        ]  # type: List[List[np.ndarray]]
```

Comment:
why not:
```
        rgb_frames = [
            [] * self.config.NUM_PROCESSES
        ]   # type: List[List[np.ndarray]]
```

Reply:
`[] * N == []` .  `[] * N` just creates an empty list.  `lst * N` just does `flatten([lst for _ in range(N)])`

Reply:
That's surprise, as well as thing above:
```
>>> k =[[]] * 10
>>> k[0].append(1)
>>> k
[[1], [1], [1], [1], [1], [1], [1], [1], [1], [1]]
```

Reply:
The `k =[[]] * 10` thing makes sense.  That is basically doing
```python
k = []
k_ele = []
for _ in range(10):
    k.append(k_ele)
```
which looks very wrong.